LayoutTests/http/wpt/html/links/redirect-301-with-fragments-test-network-cache.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE HTML>
+<html>
+<head>
+    <script src='/resources/testharness.js'></script>
+    <script src='/resources/testharnessreport.js'></script>
+    <title>Ensure fragments are correctly transferred on redirects. Disable the memory cache to ensure the network cache is being tested.</title>
+</head>
+<body>
+    <script>
+        /* Clear the memory cache and disable it */
+        internals.clearMemoryCache();
+        internals.pruneMemoryCacheToSize(0);
+
+        window.location.href = '/html/links/301-response.py#key=foo'
```

Comment:
This file needs to be in a resources/ folder or our infrastructure thinks it is a standalone test.

Reply:
Yeah, I noticed that too last night. Going to fix it today